test/e2e/clusterinfo/openshift.go

```diff
@@ -7,17 +7,23 @@ import (
 	config "github.com/openshift/api/config/v1"
 	configClient "github.com/openshift/client-go/config/clientset/versioned"
 	operatorClient "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+	routeClient "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
 	mapiClient "github.com/openshift/machine-api-operator/pkg/generated/clientset/versioned/typed/machine/v1beta1"
+	monitoringClient "github.com/prometheus-operator/prometheus-operator/pkg/client/versioned/typed/monitoring/v1"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
 	ctrlruntimecfg "sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
 // OpenShift is an Client struct which will be used for all OpenShift related functions to interact with the existing
 // Cluster.
```

Comment:
Given you are in the area, please fix the comment:
`OpenShift contains clients for interacting with the OpenShift API`